### PR TITLE
:book: extend quick start with qemu

### PR DESCRIPTION
**Pull request by mauromorales**
**What this PR does / why we need it**:

Just some improvements after having gone through the quick start

- Put in tabs the different types of machines in and how to boot on them. Bare-metal and qemu for now, can be extended with more hypervisors in the future
- Rename baremetal with bare-metal
- Add an alert for the user to be aware about the disk space needed in the prerequisites
- update the version of the iso in the example (the previous one was causing some permissions errors)
- some rewording

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

### CODE REVIEW

**Comment by mudler** [comment]
@check-spelling-bot apply [updates](https://github.com/kairos-io/kairos/actions/runs/3749021521/attempts/1).

File: docs/content/en/docs/Getting started/_index.md [hunk 1]
```diff
@@ -14,38 +14,89 @@ The same steps works on a barametal host, however, your mileage and configuratio
 
 ## Prerequisites
 
-- A VM hypervisor that boots ISOs ( or, a baremetal )
+
+{{% alert title="Warning" %}}
+Keep in mind the size of the resulting image. This will vary between distributions but you should be safe if you have 30+ Gb available disk space.
```
Comment (mudler) [inline docs/content/en/docs/Getting started/_index.md:19]: I think this is a bit misleading, which image? Maybe we should just mention disk size as requirement

File: docs/content/en/docs/Getting started/_index.md [hunk 2]
```diff
@@ -14,38 +14,89 @@ The same steps works on a barametal host, however, your mileage and configuratio
...
+{{% /alert %}}
+
+- A VM (hypervisor) or a physical server (bare-metal) that boots ISOs
 - A Linux or a Windows machine where to run the Kairos CLI (optional, we will see)
-- A `cloud-init` configuration file
+- A `cloud-init` configuration file (example below)
```
Comment (mudler) [inline docs/content/en/docs/Getting started/_index.md:25]:
```suggestion
- At least 30+ Gb of available disk space.
```

**Review by mauromorales — COMMENTED** [review]
Remove misleading alert

File: docs/content/en/docs/Getting started/_index.md (hunk 1 above)
Comment (mauromorales) [reply docs/content/en/docs/Getting started/_index.md]: good point, removed the alert all together

**Review by mudler — APPROVED** [review]
Looking good!

**Pull request merged**